### DISCUSSION TITLE
I'm having trouble getting that right now please try again later.

### DISCUSSION THREAD

**Discussion by Nakisiza** [DMcP89/harambot#72] [Q&A] [answered]
First and foremost: i added you as a collaborator as like you asked. Hopefully it helps in improving the bot.

Using commands to get the matchups or rosters from a team gives me the error in the title. I'm unsure if it has to do with the league not really starting until nba games begin and thus, week 1 actually begins or if it's related to something else. The standings command works with no problems. One question i have is: the matchup predictions are only for point based leagues or can it work with cat leagues?

**Comment by DMcP89** [comment, marked as answer]
The following issues have been opened for NBA leagues:
#76 
#74 
#73 

Will continue the troubleshooting there.
